packages/cashier/src/pages/account-transfer/account-transfer-form/account-transfer-form.tsx

```diff
@@ -122,6 +117,10 @@ const AccountTransferForm = observer(
             onChangeConverterToAmount,
             resetConverter,
         } = crypto_fiat_converter;
+        const is_open_order_position_status_present = useIsMt5LoginListStatusPresent(
+            selected_to.value ?? '',
+            'open_order_position_status' // this flag is not yet present in the BE response.
+        );
```

Comment:
I would rather go with a more generic function name and a better handler for multiple statuses rather than hard coding the value :)

Reply:
Change the comment to a TODO.

Reply:
@matin-deriv could you please suggest a function name?
and shall we define a constant like
`const Mt5LoginStatus = Object.freeze({.... FLAG_ONE:'flag_one' ......})`
and then use it here like `Mt5LoginStatus.open_order_position_status` as the second argument? 
👆🏼 hope this is the usage you have in mind.

Reply:
@matin-deriv Instead of keeping a constant file, I have implemented this hook similar to `useIsAccountStatusPresent.ts` in which the allowed flags are kept in hook itself, so we can see the available options as suggestions in code editor. Is this fine?
![Screenshot 2023-09-19 at 3 29 54 PM](https://github.com/shaheer-deriv/deriv-app/assets/122449658/32c127a6-bad3-4439-90a4-bf798b4d47e8)